index.js

```diff
@@ -30,6 +30,11 @@ module.exports = exports.default = function emailPropt({
   allowInvalidChars = false
 } = {}) {
   return new Promise((resolve, reject) => {
+    // some environments (e.g., cygwin) don't provide a tty
+    if (!process.stdin.setRawMode) {
```

Comment:
Thanks for the review! I didn't use `isatty` because I'm thinking there might be some environments where `setRawMode` might available even if the process isn't specifically a tty.